src/pamiq_core/threads/__init__.py

```diff
@@ -0,0 +1 @@
+from .thread_types import ThreadTypes
```

Comment:
@sugiyama34 素晴らしいです！タスクのこと以上のことをやっていただきありがとうございます！
その通り、ThreadTypesは`threads`モジュール以外からも利用しやすい形となると良いですね！

ただ、このままでは `ruff`に unused import errorを吐かれてしまいますし、 `threads`モジュールのtopレベルのオブジェクトである、すなわち、 `from threads import ThreadTypes`とされることを想定していることがわかりません。

よって`__all__`属性の中に格納することを推奨します。（コーディング規約的には非推奨ですが、 `from threads import *`というワイルドカードインポートができるようになります）

```suggestion
from .thread_types import ThreadTypes

__all__ = ["ThreadTypes"]
```

Reply:
いろいろ調べて理解しました！　（`__all__` は初めて知りました、、、！）
ありがとうございます！

この修正を入れようと思います、ありがとうございます！